lib/assert.js

```diff
@@ -185,6 +185,36 @@ function isArguments(object) {
   return Object.prototype.toString.call(object) == '[object Arguments]';
 }
 
+function compareMaps(a, b, strict) {
+  // check size of the maps
+  if (a.size !== b.size)
+    return false;
+
+  // iterate over all keys and check if they exist in the other
+  for (let kvs of a) {
```

Comment:
Good point. I wouldn't consider this to be hot code, but probably should stick with var for now?